src/utils/order.js

```diff
@@ -76,11 +77,23 @@ function txStatus(order: OrderType): OrderStatusType {
   return 'undefined'
 }
 
+function txStatusText(order: OrderType): string {
+  const statusTextMap = {
+    [ORDER.STATUS.WAITING_FOR_SCAN]: 'Not scanned',
+    [ORDER.STATUS.CONFIRMING]: 'Processing...',
+    [ORDER.STATUS.CONFIRMED]: moment(order.createdAt.toDate()).fromNow(),
+    [ORDER.STATUS.FAILED]: 'Transaction failed',
```

Comment:
Same as in order-row.js:  Should we move it in constants?

Reply:
same